src/main.ts

```diff
@@ -165,6 +169,45 @@ const command: Command = {
 		else {
 			return compile(config(configArgs), options);
 		}
+	},
+	eject(helper: Helper) {
+		const ejectOutput: EjectOutput = {
+			npm: {
+				devDependencies: {
```

Comment:
@agubler , these dependencies are in the `dependencies` section in `package.json` but are not being outputted as `devDependencies` of the eject command,

```
"@dojo/core": "2.0.0-alpha.20",
"@dojo/has": "2.0.0-alpha.7",
"@dojo/i18n": "2.0.0-alpha.5",
"@dojo/shim": "2.0.0-beta.8",
"cldr-data": "^30.0.1"
```

I agree that realistically, those should probably be output also...